src/main/java/com/answerdigital/benhession/academy/answerkingweek2/model/Category.java

```diff
@@ -52,53 +59,17 @@ public Category(final String name, final String description) {
         this.description = description;
     }
 
-    public void addItem(final Item item) {
-        items.add(item);
-    }
-
-    public void removeItem(final Item item) {
-        items.remove(item);
-    }
-
-    public Long getId() {
-        return id;
-    }
-
-    public void setId(final Long id) {
-        this.id = id;
-    }
-
-    public String getName() {
-        return name;
-    }
-
-    public void setName(final String name) {
-        this.name = name;
-    }
-
-    public void setDescription(final String description) {
-        this.description = description;
-    }
-
-    public String getDescription() {
-        return description;
-    }
-
-    public Set<Item> getItems() {
-        return items;
-    }
-
     @Override
     public boolean equals(final Object o) {
         if (this == o) return true;
         if (o == null || getClass() != o.getClass()) return false;
         final Category category = (Category) o;
-        return name.equals(category.name);
+        return id.equals(category.id);
```

Comment:
What's your reasoning behind changing this? I'd have argued that if a category shares the same name as another category, we'd consider the two the same.

Reply:
The reason was that id will always 100% be unique whereas name could be the same but have a different descriptions, I suppose it's debatable depending on the rest of code's funcationality.

Reply:
You make a fair point, seems we're doing it via ID in the rest of the code too. We should be able to avoid any duplicate category names because of the logic elsewhere. Will approve but keep an eye out.